types/guacamole-common-js/tsconfig.json

```
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+      "baseUrl": "../",
+      "target": "es5",
+      "module": "commonjs",
+      "noEmit": true,
+      "forceConsistentCasingInFileNames": true,
+      "noImplicitAny": true,
+      "noImplicitThis": true,
+      "strictNullChecks": true,
+      "strictFunctionTypes": true,
+      "esModuleInterop": true,
```

```suggestion
      "strictFunctionTypes": true
```
why do you need `esModuleInterop` here?
https://github.com/DefinitelyTyped/DefinitelyTyped#esmoduleinteropallowsyntheticdefaultimports

---

in the essence, why do you need a different settings here compared to one created by default?
```json
{
    "compilerOptions": {
        "module": "commonjs",
        "lib": [
            "es6"
        ],
        "noImplicitAny": true,
        "noImplicitThis": true,
        "strictFunctionTypes": true,
        "strictNullChecks": true,
        "baseUrl": "../",
        "typeRoots": [
            "../"
        ],
        "types": [],
        "noEmit": true,
        "forceConsistentCasingInFileNames": true
    },
    "files": [
        "index.d.ts",
        "***-tests.ts"
    ]
}
```